db/transaction_log_impl.cc

```diff
@@ -225,7 +225,8 @@ bool TransactionLogIteratorImpl::IsBatchExpected(
 
 void TransactionLogIteratorImpl::UpdateCurrentWriteBatch(const Slice& record) {
   std::unique_ptr<WriteBatch> batch(new WriteBatch());
-  WriteBatchInternal::SetContents(batch.get(), record);
+  Status s = WriteBatchInternal::SetContents(batch.get(), record);
+  assert(s.ok());  // TODO: What should we do with this error?
```

Comment:
At present `WriteBatchInternal::SetContents` always return `Status::OK`. So I would suggest refactoring `WriteBatchInternal::SetContents` to return `void` instead.